src/software/ai/hl/stp/tactic/shadow_enemy/simulated_shadow_enemy_tactic_test.cpp

```diff
@@ -14,12 +14,8 @@
 
 class SimulatedShadowEnemyTacticTest : public SimulatedTacticTestFixture
 {
-    void SetUp() override
-    {
-        SimulatedTacticTestFixture::SetUp();
-        setMotionConstraints({MotionConstraint::ENEMY_ROBOTS_COLLISION,
-                              MotionConstraint::ENEMY_DEFENSE_AREA});
-    }
```

Comment:
I don't think this was the problem...

Reply:
Oops, I was referencing another simulated tactic test and accidentally deleted it, I'll add it back